packages/frontend/src/commons/index.ts

```diff
@@ -37,6 +37,8 @@ export interface ChainData {
     baseTokens: Erc20Token[];
 }
 
+export const FEE_UNIT = 10000;
```

Comment:
The fee unit is actually `1_000_000`.

Reply:
Fixed in 99f113e224ae5d8f42b22a318643578a7a76414d.